save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -237,6 +243,26 @@ class DockerService(private val configProperties: ConfigProperties,
             changeOwnerRecursively(resourcesPath, "cnb")
         }
 
+        val agentPropertiesFile = File(resourcesPath, "agent.properties")
```

Comment:
May be extract this into something `fillAgentPropertiesFile()`?

Reply:
Yes, thanks, extracted